src/custom/components/AccountDetails/TransactionMod.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import styled from 'styled-components'
+import { CheckCircle, Triangle } from 'react-feather'
+
+import { useActiveWeb3React } from 'hooks'
+import { getEtherscanLink } from 'utils'
+import { ExternalLink } from 'theme'
+import { useAllTransactions } from 'state/transactions/hooks'
+import { RowFixed } from 'components/Row'
+import Loader from 'components/Loader'
+
+export const TransactionWrapper = styled.div``
+
+export const TransactionStatusText = styled.div`
+  margin-right: 0.5rem;
+  display: flex;
+  align-items: center;
+  :hover {
+    text-decoration: underline;
+  }
+`
+
+export const TransactionState = styled(ExternalLink)<{ pending: boolean; success?: boolean }>`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-decoration: none !important;
+  border-radius: 0.5rem;
+  padding: 0.25rem 0rem;
+  font-weight: 500;
+  font-size: 0.825rem;
+  color: ${({ theme }) => theme.primary1};
+`
+
+export const IconWrapper = styled.div<{ pending: boolean; success?: boolean }>`
+  color: ${({ pending, success, theme }) => (pending ? theme.primary1 : success ? theme.green1 : theme.red1)};
+`
+
+export default function Transaction({ hash }: { hash: string }) {
+  const { chainId } = useActiveWeb3React()
+  const allTransactions = useAllTransactions()
+
+  const tx = allTransactions?.[hash]
+  const summary = tx?.summary
+  const pending = !tx?.receipt
+  const success = !pending && tx && (tx.receipt?.status === 1 || typeof tx.receipt?.status === 'undefined')
```

Comment:
nit: Maybe refactor `tx.receipt?.status` into a const, so it reads easier too?

Reply:
ah shoot didnt do this, I'll do it in #60

Reply:
done in https://github.com/gnosis/gp-v2-swap/pull/60/commits/412c199951d19a204be28f667f94119665139e1f